src/cipher/encryptingstorage/tests/test_zodb.py

```diff
@@ -334,6 +371,32 @@ def untransform_record_data(self, data):
         return unhexlify(data)
 
 
+class TestServerEncryptingStorage(unittest.TestCase):
+
+    def test_load_doesnt_decrypt(self):
+        # ServerEncryptingStorage.load doesn't decrypt the record.
+        # (This prevents it from being used with a ZODB 5 Connection)
+        # See https://github.com/zopefoundation/zc.zlibstorage/issues/5
```

Comment:
This link is broken.

Reply:
No it's not, I managed to mess up copy-paste in my Chromium and somehow ended up with a U+FFFC at the end of the URL text, which was - surprise! - completely invisible in the location bar, but became visible (as %EF%BF%BC) when I copied it from the location bar.

Browsers.